drivers/adc/adc_ads1x1x.c

```diff
@@ -124,6 +126,7 @@ enum {
 
 struct ads1x1x_config {
 	struct i2c_dt_spec bus;
+	struct gpio_dt_spec alert_rdy;
```

Comment:
Shouldn't some of these need an `#ifdef CONFIG_ADC_ADS1X1X_TRIGGER` otherwise, they go unused

Reply:
i have changed it, that is consistent with other drivers